src/devices_models/devices/hydro_generation.jl

```
@@ -366,10 +382,14 @@ function add_constraints!(
     X::Type{<:PM.AbstractPowerModel},
 ) where {V <: PSY.HydroPumpedStorage, W <: AbstractHydroReservoirFormulation}
     if get_attribute(model, "reservation")
+        # TODO: Are the "reservation" models compatible with SC FF?
```

@sourabhdalvi same here

---

same response here, SC FF shouldn't be used here

---

We should throw an exception here then, same as above.